.git-blame-ignore-revs

```diff
@@ -251,3 +251,6 @@ fd14c067813572afc03ddbf7cdedc3eab5a59954
 
 # treewide format of all Nix files
 374e6bcc403e02a35e07b650463c01a52b13a7c8 # !autorebase nix-shell --run treefmt
+
+# treewide format of inactive Nix files (https://github.com/NixOS/nixpkgs/pull/364000)
+9798dad2afacacb4cfdce8dfed1493e3d4c08457
```

Comment:
This is neither a formatting commit nor a commit in any Nixpkgs branch, and all treewide formats have already been added before 😅

https://github.com/nixos/nixpkgs/commit/9798dad2afacacb4cfdce8dfed1493e3d4c08457